helm/templates/daemonset.yaml

```diff
@@ -87,8 +93,11 @@ spec:
             timeoutSeconds: 3
             periodSeconds: 2
             failureThreshold: 5
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
         - name: cs-driver-registrar
-          image: {{ printf "%s:%s" .Values.sidecars.nodeDriverRegistrarImage.repository .Values.sidecars.nodeDriverRegistrarImage.tag }}
```

Comment:
Is there a reason to move away from the printf functions for these sidecar images?  I would recommend keeping them.